engine/simulationcraft.hpp

```diff
@@ -6314,10 +6316,10 @@ struct real_ppm_t
     freq( frequency ),
     modifier( mod ),
     rppm( freq * mod ),
+    first_proc_occurred( false ),
     last_trigger_attempt( timespan_t::from_seconds( -10.0 ) ),
-    last_successful_trigger( timespan_t::from_seconds( -180.0 ) ), // Blizz done lied to us, or changed it without telling. After going through a lot of logs,
-                                                                   // it seems that it's actually 3 minutes for the precombat timer.
-    initial_precombat_time( timespan_t::from_seconds( -180.0 ) ),
```

Comment:
Is there reason to believe this comment is wrong, or the behaviour has changed? I don't know either way, but this seems to contradict the old comment.